.github/workflows/semgrep.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     container:
       image: returntocorp/semgrep
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

Comment:
Might be worth bumping to 4.1.7 / 692973e3d937129bcbf40652eb9f2f61becf3332 (not tested) to be up to date.